src/sign-verify/clu_x509_verify.c

```diff
@@ -168,8 +156,7 @@ int wolfCLU_x509Verify(int argc, char** argv)
     }
 
     if (ret == WOLFCLU_SUCCESS && caCert != NULL) {
-        if (wolfSSL_X509_LOOKUP_load_file(lookup, caCert, X509_FILETYPE_PEM)
-                != WOLFSSL_SUCCESS) {
+        if (wolfSSL_CertManagerLoadCA(cm, caCert, NULL) != WOLFSSL_SUCCESS) {
```

Comment:
I think this is the biggest change right? Is calling wolfSSL_X509_LOOKUP_load_file incorrect i.e. does it behave different then X509_LOOKUP_load_file() would if called in the same manner?

Reply:
Yes.  Note that  I have not root caused the problem.  I just noticed that if there is no lookup, (only use certmanager directly) then for some reason, certmanager will go through all the certificates in the verifyCert file.  I do not know why using the store makes a difference and IMHO I think it should not make a difference so maybe the root cause is somewhere else?